src/containers/WalletSettings/WalletSettings.vue

```diff
@@ -141,9 +156,35 @@ export default {
     hasThresholdLogged() {
       return Object.values(this.wallet).some((x) => x.accountType === ACCOUNT_TYPE.THRESHOLD)
     },
+    goToOpenloginSettingsUrl() {
+      return `${config.openLoginUrl}#clientId=${config.openLoginClientId}&redirectCode=account`
+    },
+    canShowSettingsCTA() {
+      return Object.keys(this.wallet).length === 1
```

Comment:
is this necessary? we should be able to manage openlogin for wallets with accountType threshold right?

Reply:
Yes, but right now we want to give this functionality to v2 users only.